### PR TITLE
Alterations to outlier detection and initial 3 image support

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

**Comment by kree** [comment]
Is the travis failure an off-by-one error in test_feature_extractor? 
```
  File "/home/travis/build/USGS-Astrogeology/autocnet/autocnet/matcher/tests/test_feature_extractor.py", line 27, in test_extract_features
    self.assertIn(len(features[0]), range(8,11))
AssertionError: 11 not found in range(8, 11)
```

File: autocnet/matcher/matcher.py [hunk 1]
```diff
@@ -1,13 +1,82 @@
...
+    -------
+
+    x : float
+        The x offset
+
+    y : float
+        The y offset
+
+    strength : float
+               The strength of teh correlation in the range [-1, 1].
```
Comment (jlaura) [inline autocnet/matcher/matcher.py]: learn to spell Jay

File: autocnet/matcher/matcher.py [hunk 2]
```diff
@@ -1,13 +1,82 @@
...
 
 FLANN_INDEX_KDTREE = 1  # Algorithm to set centers,
 DEFAULT_FLANN_PARAMETERS = dict(algorithm=FLANN_INDEX_KDTREE,
                                 trees=3)
 
+
+def pattern_match(template, image, upsampling=10,
+                  func=match_template):
+    """
+    Parameters
```
Comment (jlaura) [inline autocnet/matcher/matcher.py]: Add a reference here

**Comment by kree** [comment]
Again declaring a -0.7% for code coverage to be trivial and merging.

**Pull request merged**